src/components/atoms/SpacesDropdown/SpacesDropdown.tsx

```diff
@@ -10,28 +10,25 @@ import { Dropdown } from "components/atoms/Dropdown";
 
 import "./SpacesDropdown.scss";
 
+type SpaceProperty = { template?: PortalTemplate; name: string };
 export interface SpacesDropdownProps {
-  defaultSpace?: string;
-  venueId?: string;
+  defaultSpace?: SpaceProperty;
   setValue: <T>(prop: string, value: T, validate: boolean) => void;
   register: ReturnType<typeof useForm>["register"];
   fieldName: string;
   error?: FieldError;
-  venueSpaces: { template?: PortalTemplate; name: string }[];
+  venueSpaces: SpaceProperty[];
 }
 
 export const SpacesDropdown: React.FC<SpacesDropdownProps> = ({
   defaultSpace,
   venueSpaces,
-  venueId,
   setValue,
   register,
   fieldName,
   error,
 }) => {
-  const [spaceValue, setSpaceValue] = useState<string | undefined>(
-    defaultSpace
-  );
+  const [spaceValue, setSpaceValue] = useState(defaultSpace);
```

Comment:
Maybe this could be clearer:

```suggestion
  const [selected, setSelected] = useState(defaultPortal);
```

Reply:
done 6b9f914